modules/pom.xml

```diff
@@ -23,6 +23,8 @@
         <aliases.xml.basedir>src/main/aliases</aliases.xml.basedir>
         <hibernate.search.version>${project.version}</hibernate.search.version>
         <hibernate.search.module.slot>${parsedVersion.majorVersion}.${parsedVersion.minorVersion}</hibernate.search.module.slot>
+        <infinispan.integration.slot>for-hibernatesearch-${hibernate.search.module.slot}</infinispan.integration.slot>
```

Comment:
Right it's a convenience, but especially as if your app were to depend on that module you'd be forcing it to depend on a specific version of Infinispan.
By letting the module depend on it, it becomes an "implementation detail", allowing the user to use a different Infinispan version.

In this case it's a critical convenience, as Hibernate ORM will often use a different version of Infinispan for its caching purposes.. Finally, this way we can somewhat restrain the feature usage to specific versions we've tested as "fit for this purpose".